### PR TITLE
Add note about context menu to top tags options settings.

### PR DESCRIPTION
**Pull request by rdswift**
### Summary

This is a…

- [ ] Correction
- [x] Addition
- [ ] Restructuring
- [ ] Minor / simple change (like a typo)
- [ ] Other

### Reason for the Change

There was nothing in the documentation regarding the context menu items that allows showing the changed tags at the top of the list.  See the discussion in the [Community Forum](https://community.metabrainz.org/t/how-to-sort-the-tag-appearance-in-the-metadata-pane-in-picard/563528).

### Description of the Change

Add note about context menu to top tags options settings.

### Additional Action Required

Update translation files.

### CODE REVIEW

File: config/options_interface_top_tags.rst [hunk 1]
```diff
@@ -10,3 +10,11 @@ The tags specified in this option setting will always be shown in the specified
 metadata pane (which shows the metadata of selected files or tracks).
 This allows you to have the most important tags always on top of the list. Tags not listed here will be shown
 in alphabetical order below the top tags.
+
+.. note::
+
+   By default, Picard will display the tags "**Title**", "**Artist**", "**Album**", "**Track Number**", "**Length**"
+   and "**Date**" first in the list. This can be changed if you right click on one of the tags in the metadata pane,
+   and enable :menuselection:`"Show Changes First"` in the context menu.
```
Comment (phw) [inline config/options_interface_top_tags.rst:18]: I find this a bit misleading. The default of title, artist, album, ... can be directly changed with the top tags options above. "Show changes first" then is an additional option, which will then show the changes first regardless of the top tags. What about something like:

> By default, Picard will display the top tags configured here first in the list. If you enable :menuselection:`"Show Changes First"` in the context menu of the metadata pane the tags with changes will always be displayed first in the list, followed by the remaining top tags and other tags.

**Review by phw — CHANGES_REQUESTED** [review]
Yes, this definitely should get documented. I just would word it a bit differently.

File: config/options_interface_top_tags.rst (hunk 1 above)
Comment (rdswift) [reply config/options_interface_top_tags.rst:18]: Thanks for this.  I completely misunderstood, and what I had written clearly didn't make sense.  This is much better.

**Review by phw — APPROVED** [review]
Great, LGTM!

**Pull request merged**